packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1858,6 +1861,7 @@ export const MAIN_RPC_CONTEXT = {
     WEBVIEW_VIEWS_EXT: createProxyIdentifier<WebviewViewsExt>('WebviewViewsExt'),
     STORAGE_EXT: createProxyIdentifier<StorageExt>('StorageExt'),
     TASKS_EXT: createProxyIdentifier<TasksExt>('TasksExt'),
+    TASKS_MAIN: createProxyIdentifier<TasksMain>('TasksMain'),
```

Comment:
I don't see any references to this new field in the code, and I don't think a `*MAIN` entry should appear here?

Reply:
Removed.

Reply:
It looks like you've removed TASKS_MAIN from the main RPC object and left it here - it should be vice versa :-).

Reply:
Ah yes thanks. I have made the changes.